.github/workflows/int_test.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       - name: "Setup python"
         uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
+          python-version: "3.13"
```

Comment:
In a more complete IT stage, it would be good to assert success on both the lowest  and highest supported python runtimes.

Reply:
def a consideration when reworking the pipelines - probably have a choice of python version(s) + what service areas you want to test...